File: mcm-app/modules/highlight-menu/ios/HighlightMenuModule.swift
import ExpoModulesCore

public class HighlightMenuModule: Module {
  public func definition() -> ModuleDefinition {
    Name("HighlightMenu")

    View(HighlightMenuView.self) {
      Events("onHighlightRequest")

      Prop("label") { (view: HighlightMenuView, label: String) in
        view.label = label
      }

      Prop("enabled") { (view: HighlightMenuView, enabled: Bool) in
        view.menuEnabled = enabled
      }
    }
  }
}

File: mcm-app/modules/highlight-menu/android/src/main/java/expo/modules/highlightmenu/HighlightMenuModule.kt
package expo.modules.highlightmenu

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class HighlightMenuModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("HighlightMenu")

    View(HighlightMenuView::class) {
      Events("onHighlightRequest")

      Prop("label") { view: HighlightMenuView, label: String ->
        view.label = label
      }

      Prop("enabled") { view: HighlightMenuView, enabled: Boolean ->
        view.menuEnabled = enabled
      }
    }
  }
}
